x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -231,6 +232,28 @@ export function registerIndexRoutes({ router, log }: RouteDependencies) {
     })
   );
 
+  router.post(
+    {
+      path: '/internal/enterprise_search/indices/{indexName}/pipelines',
```

Comment:
@sphilipse How do I actually call this? When I tried with curl, I got a 404

Reply:
Curl should work as long as you hit the right path and have the right credentials in there. Alternatively, you can use your browser console to hit this endpoint:

```
await fetch('/xrp/internal/enterprise_search/indices', {
  method: 'POST',
  headers: {
    'Accept': 'application/json',
    'Content-Type': 'application/json',
    'kbn-xsrf': 'kibana'
  },
  body: JSON.stringify({indexName: 'jgr-le-foo', language: 'fr'})
});
```

Thirdly if you want to test the functionality and not the endpoint itself, an easy hack is to bolt it onto the addConnector functionality and just create a connector index through the UI when you want to test it.

Reply:
Ah, I bet I was leaving the base path (`/xrp/` above?) off 🤦

Reply:
I would guess so yes!